### DISCUSSION TITLE
HTTP error: 500 Internal Server Error when just uploading the first PDF doc

### DISCUSSION THREAD

**Discussion by axlgit** [jonaswinkler/paperless-ng#527] [Support] [answered]
Hi,

fresh docker installation on Debian 10. I installed docker via

 sudo docker pull jonaswinkler/paperless-ng

I had to fix the permisssions (missing +x) on manage.py in order to create my user, then I was able to login into the web interface with the credentials newly created (though it complained they were weak - for the moment it's ok), see logs in the txt file.

[manage_createsuperuser.txt](https://github.com/jonaswinkler/paperless-ng/files/5966067/manage_createsuperuser.txt)

Problem: I uploaded an invoice and when the transfer was completed I got **HTTP error: 500 Internal Server Error** 
I have only one document type, no tags or other stuff configured.

How may I fix it?

Thanks, Alex

**Comment by jonaswinkler** [comment, marked as answer]
If you just pulled the docker image:

* Paperless requires redis. The docker-compose files take care of that. That's why the recommended way to get paperless running is docker-compose. That's all described in the documentation.

If there's an internal server error, there are probably errors in the log files / in the docker log. I'll need those to see what's going on.

* The most common issue for this is that redis is not configured properly.
* You should not need to run the docker commands as root. If you do, you probably have docker configured improperly.

**Reply by AlexKalopsia** [reply]
I had the same issue, and it was a stupid mistake where I set the environment variable as `PAPERLESS_REDIS="redis://broker:6379"` instead of `PAPERLESS_REDIS=redis://broker:6379`

**Reply by axlgit** [reply]
Thanks @AlexKalopsia and @jonaswinkler , eventually I reinstalled from scratch, it was faster.
